ouroboros-network/test/Test/Ouroboros/Network/Testnet/Simulation/Node.hs

```diff
@@ -345,19 +393,26 @@ diffusion_simulation
       -> m Void
     runCommand Nothing simArgs [] = do
       threadDelay 3600
+      traceWith (diffSimTracerWithTimName (saAddr simArgs)) TrRunning
       runCommand Nothing simArgs []
-    runCommand (Just (async, _)) simArgs [] = do
-      _ <- wait async
+    runCommand (Just (_, _)) simArgs [] = do
+      -- We shouldn't block this runCommand thread waiting
+      -- on the async since this will lead to a Deadlock on
+      -- IOSim, since this returns Void.
```

Comment:
```suggestion
      -- We shouldn't block this thread waiting
      -- on the async since this will lead to a deadlock
      -- as the thread returns `Void`.
```

Reply:
done